tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "compilerOptions": {
+    "resolveJsonModule": true,
+    "noImplicitAny": false,
+    "removeComments": true,
+    "sourceMap": true
+  },
+  "extends": "@tsconfig/svelte/tsconfig.json",
+  "include": ["src/**/*", "src/node_modules"],
```

Comment:
`src` should never include `node_modules`, this looks like a mistake?

Reply:
made the suggested change as part of commit 09be8e5fdff162327be808f6336dcd762cfd9d73